### PR TITLE
Support for JanusGraph 0.2.0

### LINKED ISSUE #160: JanusGraph testing updates to reduce code copying
Released JG 0.1.1 support. now, the only bit of copied test code that remains is: 

- [x] [DynamoDBLockStoreTest](https://github.com/awslabs/dynamodb-janusgraph-storage-backend/blob/743cdb163577aad134df25a65a1d7bc45c9854d2/src/test/java/com/amazon/janusgraph/diskstorage/dynamodb/DynamoDBLockStoreTest.java#L86)
- [x] [MultiDynamoDBLogTest](https://github.com/awslabs/dynamodb-janusgraph-storage-backend/blob/5603ad301e724cf0721a3d64deeaf30e16925d99/src/test/java/com/amazon/janusgraph/diskstorage/dynamodb/MultiDynamoDBLogTest.java#L58)
- [x] [AbstractDynamoDBMultiWriteStoreTest](https://github.com/awslabs/dynamodb-janusgraph-storage-backend/blob/5603ad301e724cf0721a3d64deeaf30e16925d99/src/test/java/com/amazon/janusgraph/diskstorage/dynamodb/AbstractDynamoDBMultiWriteStoreTest.java#L252)
- [x] [AbstractDynamoDBStoreTest](https://github.com/awslabs/dynamodb-janusgraph-storage-backend/blob/5603ad301e724cf0721a3d64deeaf30e16925d99/src/test/java/com/amazon/janusgraph/diskstorage/dynamodb/AbstractDynamoDBStoreTest.java#L98)

### PR DESCRIPTION
**Pull request by amcp**
Resolves #160 #226

### CODE REVIEW

**Comment by amcp** [comment]
@sjudeng tests passed here as well. And, I was able to remove all test code. I'll push a release and merge as soon as 0.2.0 is published.

**Comment by amcp** [comment]
0.2.0 is being released as we speak. removing snapshot qualifier.

File: src/test/resources/titan-upgrade.properties [hunk 1]
```diff
@@ -15,7 +15,7 @@
 gremlin.graph=org.janusgraph.core.JanusGraphFactory
 storage.write-time=1 ms
 storage.read-time=1 ms
-storage.backend=com.amazon.janusgraph.diskstorage.dynamodb.DynamoDBStoreManager
+storage.backend=com.amazon.janusgraph.diskstorage.dynamodb.DynamoDbStoreManager
```
Comment (amcp) [inline src/test/resources/titan-upgrade.properties]: this should not change

**Comment by mattalberts** [comment]
how's the update looking?  I'm excited to pull forward and pickup the @dsl annotation additions (along with other tinkerpop updates)

**Pull request merged**